### DISCUSSION TITLE
NEXTCLOUD_MOUNT not updating

### DISCUSSION THREAD

**Discussion by r00tr4v3n** [nextcloud/all-in-one#1342] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Follow doc steps to install NC AIO
2. Add an external mount
3. docker compose up -d
 `     - NEXTCLOUD_MOUNT=/mnt/media/ # Allows the Nextcloud container to access the chosen directory on the host. See https://github.com/nextcloud/all-in-one#how-to-allow-the-nextcloud-container-to-access-directories-on-the-host`
4. Change the external mount
 `     - NEXTCLOUD_MOUNT=/mnt/data/ # Allows the Nextcloud container to access the chosen directory on the host. See https://github.com/nextcloud/all-in-one#how-to-allow-the-nextcloud-container-to-access-directories-on-the-host`
5. docker compose up -d

### Expected behavior 
The mount point should be changed or added.

### Actual behavior 
The first mount point persists.

### Host OS 
Ubuntu 20.04.4 LTS x86_64 

#### Nextcloud AIO version 
Nextcloud AIO v2.0.4
#### Current channel 
latest

**Comment by Zoey2936** [comment, marked as answer]
did you stopped the containers on the AIO panel and recreated them?

**Reply by r00tr4v3n** [reply]
No, I used portainer for that. Thank you!
In addition, the first variable was overwritten. How do you add multiple mounts?

**Reply by Zoey2936** [reply]
I think it is only possible to add one mount

**Reply by szaimen** [reply]
Simply use this then?
`- NEXTCLOUD_MOUNT=/mnt/`

**Reply by r00tr4v3n** [reply]
The mount points might be at different locations. One mount is enough for me though.

**Reply by szaimen** [reply]
> The mount points might be at different locations. One mount is enough for me though.

True, however this case is then simply not covered. Good to hear that it is enough for you!
